.travis.yml

```diff
@@ -4,6 +4,8 @@ php:
    - '7.2'
    - '7.3'
    - '7.4'
+   - '8.0'
+   - '8.1.0'
```

Comment:
```suggestion
```

Looks like Travis is not completely ready for 8.1 so we are not going to rush either.